Tests/SentryTests/Helper/SentryFileManagerTests.swift

```diff
@@ -185,7 +185,10 @@ class SentryFileManagerTests: XCTestCase {
         XCTAssertEqual(expected, fixture.delegate.envelopeItemsDeleted.invocations)
     }
 
-    func testDefaultMaxEnvelopesConcurrent() {
+    /**
+     * Disabled because flaky. Fix with https://github.com/getsentry/sentry-cocoa/issues/2017.
+     */
+    func tesDefaultMaxEnvelopesConcurrent() {
```

Comment:
Is this actually disabled because of this? I expected to see a change in an xcscheme or xcode-test.sh.

Reply:
Could you linked to a failure for us to look at?

Reply:
In any case, I only noticed this because after making some settings changes, the watchOS framework tests fail to build because we're using a watchOS ≥3.0 feature that can't compile for 2.0, and went investigating. Wonder if it's related? See `.initiallyInactive` used in `tesDefaultMaxEnvelopesConcurrent`.

Reply:
I see what you did here now, renamed it to have `tes` instead of `test` at the beginning. Please don't do this, only disable test cases via the Xcode scheme, like so:
<img width="1045" alt="image" src="https://user-images.githubusercontent.com/3241469/182003240-e65f5ae2-b531-4892-947a-8b1ddfd36356.png">

Reply:
Agreed. It took me longer than I care to admit to figure out how to reenable this test. If you disable it by renaming, please at least make it very obvious like `disabled_testDefaultMaxEnvelopesConcurrent` or something. Just removing one letter was not very obvious 😅